spec/kong_tests.conf

```diff
@@ -25,3 +25,4 @@ nginx_optimizations = off
 
 prefix = servroot
 log_level = debug
+lua_package_path=./spec/fixtures/custom_plugins/?.lua
```

Comment:
This was done to fix a failing test case in `spec/02-integration/02-cmd/02-start_stop_spec.lua:157`.